File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/__MigrationHistory.vb
Imports System.Collections.Generic

Public Class __MigrationHistory
    Public Shared Function Add(value As Models.__MigrationHistory) As Integer
        Return Data.__MigrationHistory.Add(value)
    End Function

    Public Shared Function Edit(value As Models.__MigrationHistory) As Integer
        Return Data.__MigrationHistory.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.__MigrationHistory.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.__MigrationHistory
        Return Data.__MigrationHistory.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.__MigrationHistory
        Return Data.__MigrationHistory.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.__MigrationHistory)
        Return Data.__MigrationHistory.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171036/Business/__MigrationHistory.cs
using System.Collections.Generic;

namespace Business
{
    public class __MigrationHistory
    {
	    private static Data.__MigrationHistory obj = new Data.__MigrationHistory();

        public static int Add(Models.__MigrationHistory value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.__MigrationHistory value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.__MigrationHistory GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.__MigrationHistory GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.__MigrationHistory> GetAll()
        {
            return obj.GetAll();
        }
    }
}